setup.py

```diff
@@ -21,6 +21,8 @@
     # This is needed due to https://issues.apache.org/jira/browse/BEAM-3357.
     'grpcio>=1.3.5,<=1.7.3',
     'apache-beam[gcp]>=2.2',
+    'google-api-python-client>=1.6',
```

Comment:
nit: Do you mind adding a note with a reference to Issue #71 here?

Reply:
Done.